app/build.gradle

```diff
@@ -57,7 +57,7 @@ android {
 dependencies {
     // Mapbox Navigation SDK
     implementation "com.mapbox.navigation:android:2.2.0"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.5.21"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.5.31"
```

Comment:
Let us also change this in the root build.gradle to bump to `1.5.31`

@cafesilencio